frontend/packages/kubevirt-plugin/src/components/vmtemplates/vm-template.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
+
+import {
+  TemplateSource,
+  getTemplateOperatingSystems,
+  getTemplateFlavors,
+} from 'kubevirt-web-ui-components';
+
+import { ListPage, Table, TableRow, TableData } from '@console/internal/components/factory';
+import { Kebab, ResourceLink, ResourceKebab } from '@console/internal/components/utils';
+import { VmTemplateModel } from '../../models';
+
+export const menuActions = Kebab.factory.common;
+
+const { kind, selector } = VmTemplateModel;
+
+const tableColumnClasses = [
+  classNames('col-lg-2', 'col-sm-4', 'col-xs-4'),
+  classNames('col-lg-2', 'hidden-sm', 'hidden-xs'),
+  Kebab.columnClass,
+];
+
+const VmTempleateTableHeader = () => {
+  return [
+    {
+      title: 'Name',
+      sortField: 'metadata.name',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Namespace',
+      sortField: 'metadata.namespace',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Description',
+      sortField: 'metadata.annotations.description',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Source',
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'OS',
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'Flavor',
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: '',
+      props: { className: tableColumnClasses[2] },
+    },
+  ];
+};
+VmTempleateTableHeader.displayName = 'VmTemplateTableHeader';
+
+const VmTemplateTableRow = ({ obj: template, index, key, style }) => {
+  const os = getTemplateOperatingSystems([template])[0];
+
+  return (
+    <TableRow id={template.metadata.uid} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>
+        <ResourceLink
+          kind={kind}
+          name={template.metadata.name}
+          namespace={template.metadata.namespace}
+          title={template.metadata.uid}
+        />
+      </TableData>
+      <TableData className={tableColumnClasses[0]}>
+        <ResourceLink
+          kind="Namespace"
+          name={template.metadata.namespace}
+          title={template.metadata.namespace}
+        />
+      </TableData>
+      <TableData className={tableColumnClasses[0]}>
+        {_.get(template.metadata, 'annotations.description', '-')}
```

Comment:
there is a DASH constant already in `@console/shared`

Reply:
Thanks !